hack/jenkins/job-configs/kubernetes-upgrades.yaml

```diff
@@ -56,7 +56,11 @@
         daysToKeep: 7
     builders:
         - shell: |
-            curl -fsS --retry 3  "https://raw.githubusercontent.com/kubernetes/kubernetes/{branch}/hack/jenkins/e2e.sh" | bash -
+            {provider-env}
+            {project-env}
```

Comment:
`project-env` and `job-env` are pretty ambiguous.  Add a comment?  (Project is per-upgrade-flow, job is per-step, right?)